src/Language/Haskell/Exts/InternalParser.ly

```diff
@@ -1533,6 +1533,9 @@ Template Haskell - all this is enabled in the lexer.
 >       | VARQUOTE '(' ')'              { let {l1 = $1 <^^> $3 <** [$1];
 >                                              l2 = $2 <^^> $3 <** [$2,$3];}
 >                                          in VarQuote l1 (unit_con_name l2) }
+>       | VARQUOTE '[' ']'              { let {l1 = $1 <^^> $3 <** [$1];
+>                                              l2 = $2 <^^> $3 <** [$2,$3];}
+>                                          in VarQuote l1 (list_con_name l2) }
```

Comment:
I just copied lines 1533-1535. Is there a better way to do this? I am unfamiliar with this grammar language.